content/applications/finance/payment_acquirers.rst

```diff
@@ -292,6 +292,25 @@ Payment journal
 
 The **Payment journal** selected for your payment acquirer must be a *Bank* journal.
 
+.. _payment_acquirers/publish:
+
+Publish on the website
+----------------------
+
+By default, payment acquirers are *unpublished*, which means that only internal users can see them
+on the website. Your customers can not make payments through an unpublished acquirer but they can
```

Comment:
```suggestion
on the website. Your customers cannot make payments through an unpublished acquirer but they can
```
Just nit-picking, but since I'm here might as well.